### DISCUSSION TITLE
Sony Video-Sidecar not detected

### DISCUSSION THREAD

**Discussion by mirisbowring** [immich-app/immich#9641] [Q&A]
Hi,

I have a Sony a6500 and wanted to upload the latest videos from the last holidays.
I am using the Immich Bulk Upload tool which should be able to support xmp upload:
https://github.com/immich-app/immich/discussions/2587

The camera creates the Files with the following scheme:

```
C0001.MP4
C0001M01.XML
```

I've created a script that renames all metadata sidecars to:

```
C0001.MP4
C0001.MP4.xmp
```

After the upload, I checked the video but the metadata (like location) is not there:
<img width="330" alt="image" src="https://github.com/immich-app/immich/assets/15715080/e5a0b2c1-c76a-42e5-9107-847cbc31cbac">

I checked the filesystem on the server:
<img width="392" alt="image" src="https://github.com/immich-app/immich/assets/15715080/9768f803-14c5-48fb-9213-a257c960ecbd">
As you can see, the sidecar got uploaded (yay).

How can i get Immich to recognize this sidecar?

I already executed the DISCOVER and SYNC jobs but nothing changed.

The content of the XML looks like this:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<NonRealTimeMeta xmlns="urn:schemas-professionalDisc:nonRealTimeMeta:ver.2.00" xmlns:lib="urn:schemas-professionalDisc:lib:ver.2.00" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" lastUpdate="2024-05-12T07:47:20+01:00">
	<TargetMaterial umidRef="060A2B340101010501010D431300000070BE1453420406C7B8D7AFFFFED18D25"/>
	<Duration value="1590"/>
	<LtcChangeTable tcFps="30" halfStep="true">
		<LtcChange frameCount="0" value="61563600" status="increment"/>
		<LtcChange frameCount="1589" value="47A33700" status="end"/>
	</LtcChangeTable>
	<CreationDate value="2024-05-12T07:47:20+01:00"/>
	<VideoFormat>
		<VideoRecPort port="DIRECT"/>
		<VideoFrame videoCodec="AVC_1920_1080_HP@L42" captureFps="59.94p" formatFps="59.94p"/>
		<VideoLayout pixel="1920" numOfVerticalLine="1080" aspectRatio="16:9"/>
	</VideoFormat>
	<AudioFormat numOfChannel="2">
		<AudioRecPort port="DIRECT" audioCodec="LPCM16" trackDst="CH1"/>
		<AudioRecPort port="DIRECT" audioCodec="LPCM16" trackDst="CH2"/>
	</AudioFormat>
	<Device manufacturer="Sony" modelName="ILCE-6500" serialNo="4294967295"/>
	<RecordingMode type="normal" cacheRec="false"/>
	<AcquisitionRecord>
		<Group name="ExifGPS">
			<Item name="VersionID" value="2.2.0.0"/>
			<Item name="LatitudeRef" value="N"/>
			<Item name="Latitude" value="54:39:24.808"/>
			<Item name="LongitudeRef" value="E"/>
			<Item name="Longitude" value="17:03:06.375"/>
			<Item name="TimeStamp" value="06:47:16.000"/>
			<Item name="Status" value="A"/>
			<Item name="MeasureMode" value="3"/>
			<Item name="MapDatum" value="WGS-84"/>
			<Item name="DateStamp" value="2024:05:12"/>
			<Item name="Differential" value="0"/>
		</Group>
	</AcquisitionRecord>
</NonRealTimeMeta>

```

Any Ideas?

**Comment by bo0tzz** [comment]
I don't think that's a valid XMP file

**Reply by mirisbowring** [reply]
Oh, any way that files like this will be supported in future?

How would a valid xmp look like instead?

**Reply by bo0tzz** [reply]
> How would a valid xmp look like instead?

Random example off google: https://github.com/CrossRef/pdfmark/blob/master/test-data/random-xmp-example.xmp

> any way that files like this will be supported in future?

It looks like some custom format that sony came up with so I don't know if it's worth it for us to support that.
